shared/api/cluster_state.go

```diff
@@ -15,6 +15,7 @@ type ClusterMemberSysInfo struct {
 	TotalSwap    uint64    `json:"total_swap" yaml:"total_swap"`
 	FreeSwap     uint64    `json:"free_swap" yaml:"free_swap"`
 	Processes    uint16    `json:"processes" yaml:"processes"`
+	NumCpu       int       `json:"num_cpu" yaml:"num_cpu"`
```

Comment:
Go has a convention that fields/variables should use uppercase characters for acronyms. 

So this should be NumCPU.

Reply:
Changed the field name.